### ISSUE TITLE
Add dataset: us_national_archives_flickr

### ISSUE THREAD

**Issue by nabsiddiqui** [bigscience-workshop/lam#75]
### A URL for this dataset

https://www.flickr.com/photos/usnationalarchives
### Dataset description

Dataset of images from the US National Archives, largely taken from the Women's Bureau and the Environmental Protection Agency. More information can be found in the About section of the Flickr page along with the FAQ on the National Archive's website: https://www.archives.gov/social-media/flickr-faqs.html

### Dataset modality

Image

### Dataset licence

No known copyright restrictions: https://www.archives.gov/social-media/flickr-faqs.html#9

### How can you access this data

As a download from a repository/website

### Confirm the dataset has an open licence

- [X] To the best of my knowledge, this dataset is accessible via an open licence

**Comment by davanstrien** [comment]
Is this a duplicate of #72?

**Comment by nabsiddiqui** [comment]
Sorry I wrote the title wrong. No it isn't a duplicate. These are images by the US National Archives. They are also on Flickr. I don't know if I can change the title. Can you change it or recreate the issue with the correct title? I'm not at my computer.

**Comment by davanstrien** [comment]
I've updated the title, think you should also be able to modify it if needed.

**Comment by nabsiddiqui** [comment]
@davanstrien Can you add this to the project board so I can claim it and upload it? I have the images and metadata downloaded and cleaned now: https://github.com/orgs/bigscience-workshop/projects/8/views/1?query=is%3Aopen+sort%3Aupdated-desc

**Comment by nabsiddiqui** [comment]
#self-assign

**Comment by nabsiddiqui** [comment]
Ok I have uploaded this now.

**Comment by davanstrien** [comment]
Thanks for working on this. Would you be able to add a little bit of documentation describing the dataset. Writing a full datacard would be great but even a high level summary of the data would be great so other people know what the data is, how to use it and potential biases and limitations in the collection.

**Comment by nabsiddiqui** [comment]
Yes I will add some information, but this is the first week of class, so I don't know when I will be able to get to it. Hopefully by next week.
